src/components/dictionaryConcepts/containers/CreateConcept.jsx

```diff
@@ -77,8 +78,10 @@ export class CreateConcept extends Component {
         params: { collectionName, type, typeName },
       },
     } = this.props;
-    if (Object.keys(nextProps.newConcept).length) {
-      nextProps.history.push(`/concepts/${type}/${typeName}/${collectionName}`);
+    if (Object.keys(nextProps.newConcept).length && nextProps.addedConcept.length) {
```

Comment:
Hey, Emma, would you mind if you put "Object.keys(nextProps.newConcept).length" and "nextProps.addedConcept.length"  in separate variables and use them in the if statement. Quite sure would help make the code easy to read.

Reply:
Done